workflow/agents/examples/overview.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: "Overview"
+---
+
+Explore our collection of examples to learn how to build robust agent systems. From architectural patterns to real-world implementations, these examples demonstrate the full potential of our Agents API.
+
+## Architectures & Patterns
```

Comment:
I don't think we need another page. We already have these cards on [/workflow/agents/overview](https://upstash-dx-1605.mintlify.app/workflow/agents/overview)